pomSnapshotToGitHash.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/sh 
```

Comment:
i don't see how these files are run.  do we need an entry in .travis.yml to execute these bash scripts?

Reply:
Correct. Will add.